### PR TITLE
devnet: Delete unused flags

### PR DESCRIPTION
**Pull request by trianglesphere**
**Metadata**

- Fixes CLI-3888

### CODE REVIEW

File: ops-bedrock/docker-compose.yml [hunk 1]
```diff
@@ -98,8 +98,6 @@ services:
       OP_PROPOSER_ROLLUP_RPC: http://op-node:8545
       OP_PROPOSER_POLL_INTERVAL: 1s
       OP_PROPOSER_NUM_CONFIRMATIONS: 1
-      OP_PROPOSER_SAFE_ABORT_NONCE_TOO_LOW_COUNT: 3
-      OP_PROPOSER_RESUBMISSION_TIMEOUT: 30s
```
Comment (refcell) [inline ops-bedrock/docker-compose.yml:102]: These flags are part of the `txmgr`'s cli flags [here](https://github.com/ethereum-optimism/optimism/blob/f9928f1e3885bd556171f541046001a6278537d9/op-service/txmgr/cli.go#L76-L87) I believe. Should be ok to delete but just wanted to point out I think they are used, just optional.

File: ops-bedrock/docker-compose.yml [hunk 2]
```diff
@@ -125,15 +123,9 @@ services:
       OP_BATCHER_ROLLUP_RPC: http://op-node:8545
       OFFLINE_GAS_ESTIMATION: false
       OP_BATCHER_MAX_CHANNEL_DURATION: 1
-      OP_BATCHER_MAX_L1_TX_SIZE_BYTES: 120000
-      OP_BATCHER_TARGET_L1_TX_SIZE_BYTES: 100000
-      OP_BATCHER_TARGET_NUM_FRAMES: 1
-      OP_BATCHER_APPROX_COMPR_RATIO: 0.4
```
Comment (refcell) [inline ops-bedrock/docker-compose.yml:131]: ibid

File: ops-bedrock/docker-compose.yml [hunk 3]
```diff
@@ -125,15 +123,9 @@ services:
...
       OP_BATCHER_SUB_SAFETY_MARGIN: 4 # SWS is 15, ChannelTimeout is 40
       OP_BATCHER_POLL_INTERVAL: 1s
       OP_BATCHER_NUM_CONFIRMATIONS: 1
-      OP_BATCHER_SAFE_ABORT_NONCE_TOO_LOW_COUNT: 3
-      OP_BATCHER_RESUBMISSION_TIMEOUT: 30s
```
Comment (refcell) [inline ops-bedrock/docker-compose.yml:136]: ibid

File: ops-bedrock/docker-compose.yml (hunk 1 above)
Comment (trianglesphere) [reply ops-bedrock/docker-compose.yml:102]: yea. All of these flags should be defaulted.

**Review by protolambda — APPROVED** [review]

**Pull request merged**